CMakeLists.txt

```diff
@@ -1,192 +1,94 @@
+find_package(TIFF REQUIRED)
 find_package(PNG REQUIRED)
-find_package(ZLIB REQUIRED)
-find_package(X REQUIRED)
-
-set(rsxs_dir lib/rsxs-1.0)
-
-include_directories(${OpenGL_INCLUDE_DIR}
-                    ${PNG_INCLUDE_DIR}
-                    ${KODI_INCLUDE_DIR}/.. # Hack way with "/..", need bigger Kodi cmake rework to match right include ways
-                    ${X_INCLUDE_DIRS}
-                    ${CMAKE_BINARY_DIR}/rsxs-prefix/src/rsxs-build
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/lib
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/cyclone
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/euphoria
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/fieldlines
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/flocks
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/flux
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/helios
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/hyperspace
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/lattice
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/plasma
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/skyrocket
-                    ${PROJECT_SOURCE_DIR}/${rsxs_dir}/src/solarwinds)
-                        
-
-get_filename_component( COMPILER_FILENAME "${CMAKE_C_COMPILER}" NAME )
-string( REGEX REPLACE "-[^-]+$" ""
-        TOOLCHAIN_NAME "${COMPILER_FILENAME}" )
-
-include(ExternalProject)
-set(update_command "")
-if(BOOTSTRAP_IN_TREE OR NOT DEFINED BOOTSTRAP_IN_TREE)
-  set(update_command UPDATE_COMMAND autoreconf -vif WORKING_DIRECTORY <SOURCE_DIR>)
-endif()
-set(configure_start ${PROJECT_SOURCE_DIR}/${rsxs_dir}/configure)
-if(DEPENDS_PATH)
-  set(PNG_PATH "--with-png=${DEPENDS_PATH}")
-  if(EXISTS ${DEPENDS_PATH}/share/config.site)
-    set(configure_start CONFIG_SITE=${DEPENDS_PATH}/share/config.site ${configure_start})
-  endif()
+find_package(LTDL)
+
+# set(APP_RENDER_SYSTEM "gles") # Leaved here for test purpose only
+
+if(APP_RENDER_SYSTEM STREQUAL "gl" OR NOT APP_RENDER_SYSTEM)
+  find_package(OpenGl REQUIRED)
+  set(DEPLIBS ${OPENGL_LIBRARIES})
+  set(includes ${OPENGL_INCLUDE_DIR})
+  add_definitions(${OPENGL_DEFINITIONS})
+elseif(APP_RENDER_SYSTEM STREQUAL "gles")
+  find_package(OpenGLES REQUIRED)
+  set(DEPLIBS ${OPENGLES_LIBRARIES})
+  set(includes ${OPENGLES_INCLUDE_DIR})
+  add_definitions(${OPENGLES_DEFINITIONS})
 endif()
-if(APPLE)
-  set(EXTRA_FLAGS "CFLAGS=${CMAKE_C_FLAGS} -fPIC -isystem${DEPENDS_PATH}/include -isystem${KODI_INCLUDE_DIR}" "CXXFLAGS=${CMAKE_CXX_FLAGS} -fPIC -isystem${DEPENDS_PATH}/include -isystem${KODI_INCLUDE_DIR}")
+
+include_directories(${KODI_INCLUDE_DIR}/.. # Hack way with "/..", need bigger Kodi cmake rework to match right include ways
+                    ${GLM_INCLUDE_DIR}
+                    ${GLI_INCLUDE_DIR})
+
+if(ImageMagick_FOUND AND NOT ${OWN_IMAGE_MAGICK})
```

Comment:
Why is this needed?

Reply:
This was thought as to make a quik linux build with only a `cmake ..` for test ImageMagick and handle his configure command, was easier as over the depends build and thought to leave as fallback if someone compile it itself.

But everything related ImageImagick becomes removed, after next request no more needed.